bookings/components/booking-filters.yaml

```diff
@@ -0,0 +1,464 @@
+    bring back bookings between 1 to 4 *and* between 10 to 15.
+  example: 1-10
+assigned_area:
+  name: assigned_area
+  in: query
+  required: false
+  explode: false
+  schema:
+    type: array
+    items:
+      type: string
+  description: >-
+    Filters on bookings that are assigned to the given area ID. If a comma-separated list is passed, 
+    bookings assigned to any of the given area IDs will be returned. Alternatively, to filter on 
+    whether bookings have an assigned area or not, use `true` or `false`. 
+  example: 5cab10d8f44d3861c8c3819d
+assigned_to:
+  name: assigned_to
+  in: query
+  required: false
+  explode: false
+  schema:
+    type: array
+    items:
+      type: string
+  description: >-
+    Filters on bookings that are assigned to the given user ID. If a comma-separated list is passed, 
+    bookings assigned to any of the given user IDs will be returned. To filter on bookings that are 
+    not assigned to a user, use `none`.
+  example: 5cab111386510d011b597b3f
+reference:
+  name: reference
+  in: query
+  required: false
+  schema:
+    type: integer
+  example: 12345
+  description: >-
+    Filters on bookings that have the given booking reference ID.
+created_date:
+  name: created_date
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-01-01'
+  description: >-
+    Filters on bookings that were created on the given date.
+created_date_from:
+  name: created_date_from
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-01-01'
+  description: >-
+    Filters on bookings that were created on or after the given date.
+created_date_to:
+  name: created_date_to
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-03-01'
+  description: >-
+    Filters on bookings that were created on or before the given date
+current_stage:
+  name: current_stage
+  in: query
+  required: false
+  schema:
+    type: string
+  example: Arrived
+  description: >-
+    Filters on bookings that have the given `current_stage`.
+date:
+  name: date
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-01-01'
+  description: >-
+    Filters on bookings that take place on the given date.
+date_from:
+  name: date_from
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-01-01'
+  description: >-
+    Filters on bookings that take place on or after the given date.
+date_to:
+  name: date_to
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-01-01'
+  description: >-
+    Filters on bookings that take place on or before the given date.
+email:
+  name: email
+  in: query
+  required: false
+  schema:
+    type: string
+  example: test@example.com
+  description: >-
+    Filters on bookings that have the given email address.
+exclude_auto_confirmed:
```

Comment:
It's not a docs change, but do you think the two `exclude_` filters should be renamed without the 'exclude' eg `auto_confirmed: true|false`? That way it's consistent with how others like `preorders` work

Reply:
💯We've got some reports that rely on the `exclude_` filters atm, but we could add the additional filters and change the documentation to not mention the `exclude_` ones?